### ISSUE TITLE
Improve learning rate annealing for faster training of neural networks

### ISSUE THREAD

**Issue by raspstephan** [raspstephan/CBRAIN-Keras-Diagnostics#1]
this is closely related to #7

There are ways to train neural networks significantly faster than just using standard learning rate annealing. I heard about this from the fast.ai work on the DAWNBench competition: https://www.fast.ai/2018/08/10/fastai-diu-imagenet/

I am not as firm on the backgrounds yet, but I think a large part of the training speedup comes from implementing Leslie Smith's work on Super-convergence (https://arxiv.org/abs/1708.07120) and the 1Cycle learning rate scedule (https://arxiv.org/abs/1803.09820). 

I am not sure how easy this would be to implement but there is a Keras repository: https://github.com/titu1994/keras-one-cycle

This links
